qml/Sailfish/Silica/ListItem.qml

```diff
@@ -9,8 +9,23 @@ SimpleRow {
 
     height: contentHeight
 
-    // HACK: Display arrow on the bottom right
-    text: " "
-    detailText: " "
+    // Do not show right angle in iOS
```

Comment:
This comes from FontAwesome: They also have "arrow right", which is something different to "angle right".